### DISCUSSION TITLE
Problem with aosp

### DISCUSSION THREAD

**Discussion by xerel89** [LSPosed/LSPosed#2056] [Q&A]
Hey

I downloaded the aosp module and installed with magisk but lsposed can not detected the module!
What I make wrong?
My device is a redmi note 9 pro using lineage 18.1...

And yes I use the actually version from magisk and lsposed !

![Screenshot_20220730-132928_Settings.png](https://user-images.githubusercontent.com/33770412/181909065-b715e688-9d51-4c85-bc91-bfb8aa9b2a77.png)
![Screenshot_20220730-132906_Shell.png](https://user-images.githubusercontent.com/33770412/181909085-6da5ff74-30dc-4776-926e-d2f34b74cfa6.png)
![Screenshot_20220731-194704_Shell.png](https://user-images.githubusercontent.com/33770412/182038938-7d446b5a-39a0-40ad-9608-147a8d207969.png)

[magisk log details](https://mega.nz/file/fGoXxTTQ#u0UJCSj9A_2Kv9mFtjgYWO-lXYtKvXnnFzBM2dUl2I8)
[lsposed log details](https://mega.nz/file/ObI0UQZA#UnkptndWqKzZdMkV8oWoiu6v4_iB-Q_YIsl8VoCk4uw)

**Comment by Originalimoc** [comment]
Looks like installed successfully to me?
What's your problem exactly?

**Reply by xerel89** [reply]
Yes magisk I have no problem with the installation, the bug is integrated in lsposed

**Comment by xerel89** [comment]
So I have make now again an upload from the lsposed log [here](https://mega.nz/file/ObI0UQZA#UnkptndWqKzZdMkV8oWoiu6v4_iB-Q_YIsl8VoCk4uw)

And when I am touch on the memory card button open the mixplorer is that normal?

**Comment by siavash79** [comment]
Please also share a screenshot from the puzzle icon (with badge 8)

**Reply by xerel89** [reply]
I have the what's app module uninstalled, but I it's to buring for me, so I that's is the reason why are only anymore seven modules are installed :)

**Reply by xerel89** [reply]
And will you come out of logs smart or do they remain a mystery to you?

**Reply by tijekgofiyab** [reply]
hello . same issue happened to me and  i just found a solution go on your magisk home screen and direct install magisk

**Reply by tijekgofiyab** [reply]
reboot phone flash magiskhide props config and then flash lsposed done
